src/scene/SceneManager.js

```diff
@@ -1032,6 +1032,19 @@ var SceneManager = new Class({
 
         if (!scene)
         {
+            var isFound = false;
+            for (var i = 0; i < this._pending.length; i++)
+            {
+                if (this._pending[i].key === key)
+                {
+                    isFound = true;
+                    break;
+                }
+            }
+            if (isFound)
+            {
+                this.queueOp('start', key, data);
```

Comment:
This line could be moved in place of line 1040, `isFound = true;`. Avoids the extra conditional and variable. :)

Reply:
thanks! good catch